persist-cli-tests/src/test/resources/test-src/output/tool_test_db_push_10/persist/entities_db_script.sql

```diff
@@ -5,8 +5,8 @@ CREATE TABLE Profile (
 	id INT NOT NULL,
 	name VARCHAR(191) NOT NULL,
 	isAdult BOOLEAN NOT NULL,
-	salary FLOAT NOT NULL,
-	age DECIMAL NOT NULL,
+	salary DOUBLE NOT NULL,
```

Comment:
What is the default precision for DOUBLE? is it the maximum of 64?

Reply:
No, the precision value is from 24 to 53(https://dev.mysql.com/doc/refman/8.0/en/floating-point-types.html).

But, Ballerina float is the same as double in Java(https://ballerina.io/learn/by-example/floating-point-numbers/). Java double is mapped with MySQL Double(https://docs.oracle.com/cd/E19501-01/819-3659/gcmaz/).

Reply:
Seems like it wont be an error for precision mismatch, the value would be rounded up. Seems the usual practice with MySQL double data type.